From f5550422e202041a70cce8776785e3e651a01d8d Mon Sep 17 00:00:00 2001
From: Philipp Kaluza <phk@src.gnome.org>
Date: Thu, 03 Jan 2013 10:51:19 +0000
Subject: fix build error due to missing gweather-xml.h

GWeather doesn't export the deprecated GWeatherXML anymore since
commit 354f1aa9003ee08c6e63ecbaaa6b9179b9178184 . Luckily, our last
usage of it is long gone. Remove leftover include.

See https://bugzilla.redhat.com/show_bug.cgi?id=890219
---
diff --git a/applets/clock/clock.c b/applets/clock/clock.c
index 90ee62b..6b62daa 100644
--- a/applets/clock/clock.c
+++ b/applets/clock/clock.c
@@ -54,7 +54,6 @@
 #define GNOME_DESKTOP_USE_UNSTABLE_API
 #include <libgnome-desktop/gnome-wall-clock.h>
 
-#include <libgweather/gweather-xml.h>
 #include <libgweather/location-entry.h>
 #include <libgweather/timezone-menu.h>
 #include <libgweather/gweather-enum-types.h>
--
cgit v0.9.1
